docs/konnectors.md

```diff
@@ -320,6 +320,56 @@ On the host create the file /etc/rkt/net.d/10-containers.conf
 
 and run your container with the "--net=bridge" option. That way, a new interface is available in the container and gives you access to the host.
 
+## Multi-account handling
+
+This section is devoted to allow the user to use one account for multiple konnectors. It will
+follow the following constraints in mind:
+
+- The migration path must be as easy as possible
+- The developpement and maintainance of konnector must also be as easy as possible
+
+### New doctype : io.cozy.konnector_account or io.cozy.account
```

Comment:
io.cozy.accounts seems to be a good candidate but this won't be the same as the email application. I suppose this does not cause any problem since the email app won't be ported to V3 in a near future.